### ISSUE TITLE
EasyWindowDrag stops working "sometimes"

### ISSUE THREAD

**Issue by vladdeSV** [vladdeSV/my-ahk-scripts#2]
As title suggests. It stops working sometimes, and I do not know why,

**Comment by vladdeSV** [comment]
I believe all scripts stopped working periodically sometime ago, but I could not figure out why. I made a blind guess that it was related to c5141660f0f9fd34449c0be2fd3d971d965e5bfa, so I temporarily disabled them in cf1645b3f33c67046fd535da506c57caaa7817fd.

Today I verified that both `__main.ahk2` and `EasyWindowDrag.ahk2` were running, but the latter did not respond to commands.

This might have helped, or it might have remedied a completely whole other problem.
